RACK-Ontology/ontology/ANALYSIS.sadl

```diff
@@ -10,32 +10,44 @@
  * of the Defense Advanced Research Projects Agency (DARPA).
  */
 
-uri "http://arcos.rack/ANALYSIS" alias ^A.
+
+/************** edit history *****************
+  *
+  * dwa - 11 Nov 2020 - changed ANALYSIS_REPORT to a kind of REPORT vs. ENTITY, since REPORT is defined already
+  * dwa - 11 Nov 2020 - minor updates to note fields
+  * dwa - 11 Nov 2020 - reduced cardinality of result in ANALYSIS_REPORT to 1, since N seems ill-defined
+  * dwa - 11 Nov 2020 - reduced cardinality of ANALYSIS_REPORT in ANALYSIS_ANNOTATION to 1, since an annotation will naturally come from a single report
+  * dwa - 13 Nov 2020 - added import for DOCUMENT to allow ANALYSIS_REPORT to be a kind of REPORT
+  */
+  
+uri "http://arcos.rack/ANALYSIS" alias ^An.
 import "http://arcos.rack/PROV-S".
+import "http://arcos.rack/DOCUMENT".
 
 ANALYSIS_REPORT
-	(note "Output of an ANALYSIS")
-	is a type of ENTITY.
+    (note "A document that is the output of an ANALYSIS")
+    is a type of REPORT.
```

Comment:
I don't think that the note should be changed and  this should be a subclass of the Document Report. I would actually expect a Document Report to contain a collection of ANALYSIS_REPORTs, which describe a single analysis, An example may be a code coverage report where each line of code is is tracked as hit or not. Then each of these individual reports would be aggregated into a full REPORT document.  The ANALYSIS_REPORT is intended what is produced by the ANALYSIS, similar to a REVIEW and a REVIEW_LOG.

Maybe we should change the ANALYSIS_REPORT to some else like a ANALYSIS_STATEMENT or something along those lines. This would also allow us to remove the document import.

Reply:
OK, so I think the distinction is that we're thinking of the analysis output as simple computer-generated output and not a curated, versioned document kind of thing

Reply:
Yes.  Maybe ANALYSIS_OUTPUT would be the better term.